vars/helm.groovy

```diff
@@ -78,10 +80,15 @@ def deployRemoteChart(credentialsId, environment, namespace, chartName, chartVer
       file(credentialsId: "$chartName-$environment-values", variable: 'values')
     ]) {
       def extraCommands = getExtraCommands(chartVersion)
-      sh "helm repo add ffc $HELM_CHART_REPO"
-      sh "helm repo update"
+      addHelmRepo('ffc', $HELM_CHART_REPO)
```

Comment:
Am assuming here that `$HELM_CHART_REPO` points to the gitlab location of the demo service helm charts which is different to the location of the `ffc-helm-library` stored on github. The same name `ffc` is used for both. Whilst this won't cause any issues as it stands I think it's confusing to use the same name for two different repos. I would suggest a different name.